### PR TITLE
Make ISAR compatible with Python 3.9 and 3.10 again

### PR DESCRIPTION
**Pull request by tsundvoll**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,14 +7,15 @@ name = "isar"
...
-requires-python = ">=3.11"
+requires-python = ">=3.9"
 license = { file = "LICENSE" }
 classifiers = [
     "Intended Audience :: Developers",
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: Eclipse Public License 2.0 (EPL-2.0)",
     "Natural Language :: English",
     "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
```
Comment (Christdej) [inline pyproject.toml:18]: Should Python :: 3.10 also be included here?

Comment (tsundvoll) [reply pyproject.toml:18]: Thanks!

**Review by Christdej — APPROVED** [review]
LGTM

**Comment by tsundvoll** [comment]
This still need to be verified together with Flotilla

**Comment by tsundvoll** [comment]
> This still need to be verified together with Flotilla

Verified ✅

**Pull request merged**